packages/shared/src/contexts/payment/StoreKit.tsx

```diff
@@ -50,8 +54,15 @@ export const StoreKitSubProvider = ({
 
   const { data: productOptions } = useQuery({
     queryKey: ['iap-products'],
-    enabled: !!productIds,
+    enabled:
+      !!productIds &&
+      messageHandlerExists(WebKitMessageHandlers.IAPSubscriptionRequest) &&
+      !!user?.isTeamMember,
```

Comment:
Just to make sure we don't show the link before we're ready and have tested it in production.